### PR TITLE
LTS 2.0 Launch

### PR DESCRIPTION
**Pull request by Skptak**
LTS 2.0 Retag
Description
-----------
Fixing some small MISRA related issues, and then getting everything ready for LTS 2.0

Checklist:
----------
- [ x ] I have tested my changes. No regression in existing tests.
- [ x ] My code is formatted using Uncrustify.
- [ x ] I have read and applied the rules stated in CONTRIBUTING.md. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tools/coverity/misra.config [hunk 1]
```diff
@@ -38,6 +42,10 @@
             deviation: "Rule 8.9",
             reason: "For ease, configuration parameters are defined at a global scope even when used only once."
         },
+        {
+            deviation: "Rule 8.13",
```
Comment (gitwallit) [inline tools/coverity/misra.config:46]: did you have a look at how this rule has been successfully suppressed inline in the +tcp library - https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/dcd0df708af3083d9a72bf3154caf038dacf6dbd/source/FreeRTOS_IP.c#L235

File: tools/coverity/misra.config [hunk 2]
```diff
@@ -30,6 +30,10 @@
             deviation: "Rule 3.1",
             reason: "Allow nested comments. Documentation blocks contain comments for example code."
         },
+        {
+            deviation: "Rule 8.6",
```
Comment (gitwallit) [inline tools/coverity/misra.config:34]: could you discuss with Aniruddha/Alfred. They also ran into similar issues with incline suppression.

**Review by gitwallit — DISMISSED** [review]

File: tools/coverity/misra.config (hunk 2 above)
Comment (Skptak) [reply tools/coverity/misra.config:34]: I messaged him about this offline. Marking as resolved.

File: tools/coverity/misra.config (hunk 1 above)
Comment (Skptak) [reply tools/coverity/misra.config:46]: I messaged him about this offline. Marking as resolved.

**Review by AniruddhaKanhere — DISMISSED** [review]

**Review by gitwallit — DISMISSED** [review]

**Review by AniruddhaKanhere — DISMISSED** [review]

**Review by jasonpcarroll — APPROVED** [review]

**Review by archigup — APPROVED** [review]

**Pull request merged**